pkg/machinery/config/types/v1alpha1/v1alpha1_features.go

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package v1alpha1
+
+// RBACEnabled implements config.Features interface.
+func (f *FeaturesConfig) RBACEnabled() bool {
+	if f.RBAC == nil {
```

Comment:
we should also generate new config for Talos v0.11 with RBAC enabled by default (that's our goal anyways)

Reply:
Done in that PR